frontend/src/ui/components/ToggleSwitch/ToggleSwitch.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+import './styles.scss';
+
+interface ToggleSwitchProps {
+  fieldName: string;
+  initialValue?: boolean;
+  onChange?: any;
+  error?: string;
+  disabled?: boolean;
+}
+
+export const ToggleSwitch: React.SFC<ToggleSwitchProps> = (
+  props: ToggleSwitchProps
+) => {
+  const [value, setValue] = React.useState(props.initialValue);
+
+  // Ensure the correct value is shown if props are updated.
+  React.useEffect(() => {
+    setValue(props.initialValue || false);
+  }, [props.initialValue]);
+
+  const onChange = () => {
+    props.onChange(props.fieldName, !value);
+  };
+
+  return (
+    <label className="switch">
+      <input
```

Comment:
@Agrendalath Can you use the react boostrap default switch (https://react-bootstrap.github.io/components/forms/#forms-custom-switch)?

It also looks too big and misaligned for the rest of the text (the one from bootstrap handles scaling automatically).

![image](https://user-images.githubusercontent.com/27893385/78143563-d296a280-7404-11ea-943d-ed29247b0767.png)